tests/Doctrine/Tests/ORM/Functional/SchemaTool/SqliteSchemaToolTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\SchemaTool;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Annotation as ORM;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\SchemaTool;
+use Doctrine\Tests\OrmTestCase;
+use Doctrine\Tests\TestUtil;
+
+class SqliteSchemaToolTest extends OrmTestCase
```

Comment:
I think that using the `OrmFunctionalTestCase` as parent class would simplify this class a bit (remove the need for the properties and the setup method)

Reply:
I get a bunch of errors in other functional tests when I use OrmFunctionalTest.  
Looks like they are used some dirty global state... 
It was only a reason to use OrmTestCase.
I'll try, ok.